src/KygekDev/Utils/utils/DiskUsage.php

```diff
@@ -5,17 +5,17 @@
 namespace KygekDev\Utils\utils;
 
 use pocketmine\command\CommandSender;
-use pocketmine\command\PluginCommand;
+use pocketmine\command\Command;
 use pocketmine\plugin\Plugin;
 use pocketmine\Server;
 use pocketmine\utils\TextFormat as TF;
 use RecursiveDirectoryIterator as RDI;
 use RecursiveIteratorIterator as RII;
 
-class DiskUsage extends PluginCommand {
+class DiskUsage extends Command {

```

Comment:
Could you also implement the `PluginOwned` interface?

Reply:
I don't know. Can you make a suggestion?
I found it working fine so I created a PR...

Reply:
Here's an example:
https://github.com/KygekTeam/KygekRanksUI/blob/pm4/src/Kygekraqmak/KygekRanksUI/Commands.php